pkg/provisioner/ironic/ironic.go

```diff
@@ -248,6 +248,13 @@ func (p *ironicProvisioner) ValidateManagementAccess(credentialsChanged bool) (r
 		}
 		p.publisher("Registered", "Registered new host")
 
+		raidInterface := p.bmcAccess.RAIDInterface()
+		if raidInterface != "" {
```

Comment:
The type of the RAIDInterface member in nodes.CreateOpts is string, so it will be initialised to an empty string on lines 239-244 anyway. There's no harm in just passing

    RAIDInterface: p.bmcAccess.RAIDInterface(),

after line 241. You don't need to do anything special to check if the BMC plugin is returning an empty string.

Reply:
Thanks @zaneb . You're right.
But it seems that these Interfaces already added by [1]
So I would like to configure the RAID interface to `irmc` in this PR :)

[1] https://github.com/metal3-io/baremetal-operator/commit/c8fdf0483ec30bd1fb9154ba8cc291e340d10f06#diff-c3f8fb66d64901a306b6faf6f7019c9f